docs/CONTRIBUTING.md

```diff
@@ -54,6 +54,8 @@ submitting a PR:
 - PR titles indirectly become part of the CHANGELOG so it's crucial to provide a good record of **what** change is being
   made in the title; **why** it was made will go in the PR description, along with a link to a GitHub issue if it
   exists.
+- Please start the commit message and PR title with the string **[breaking]** if the PR contains a breaking change. A
```

Comment:
```suggestion
- If the PR contains a breaking change, please start the commit message and PR title with the string **[breaking]**. A
```
I think this wording improves readability because it provides the condition up front, so it's easier to understand the purpose of the bullet point when skimming the document. That makes it easier for those who need it to find, and faster for those who don't need it to skip over.